resources/sdk/clisdkclient/templates/root.mustache

```diff
@@ -35,9 +48,52 @@ var versionCmd = &cobra.Command{
 	Long:  `All software has versions. This is gc version's`,
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println("{{packageVersion}}")
+
+        var latestVersion string
+        if strings.Contains("{{basePath}}", "api.mypurecloud.com") {
+            latestVersion = retrieveLatestVersion(externalVersionURL)
+        } else {
+            latestVersion = retrieveLatestVersion(internalVersionURL)
+        }
+
+        if latestVersion == "{{packageVersion}}" {
+            fmt.Println("You're all up to date.")
+        } else {
+            fmt.Printf("New version of the CLI is available: %v\n", latestVersion)
+        }
 	},
 }
 
+func retrieveLatestVersion(url string) string {
+	client := http.Client{
+		Timeout: time.Second * 2,
+	}
+	req, err := http.NewRequest(http.MethodGet, url, nil)
+	if err != nil {
+		logger.Fatal(err)
```

Comment:
It's probably best to have retrieveLatestVersion return an error and a string and return the error and an empty string if an error is encountered in the function.
Then have the caller check if the error isn't nil and print something like "error retrieving latest version".